Use Math Methods/app.js

```diff
@@ -0,0 +1,4 @@
+const temperature = 37.5;
+
+const tempRounded = Math.round(temperature);
+const tempRoundDown = Math.floor(temperature);
```

Comment:
Hi @ct-aa, there are more Math methods that you can use aside from `round` and `floor`. You may check them here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math